emails/apps.py

```diff
@@ -48,8 +48,8 @@ def __init__(self, app_name, app_module):
 
         # badwords file from:
         # https://www.cs.cmu.edu/~biglou/resources/bad-words.txt
```

Comment:
```suggestion
        # https://www.cs.cmu.edu/~biglou/resources/bad-words.txt
        # Using `.text` extension because of https://github.com/dependabot/dependabot-core/issues/1657
```